Tasks/google-play-release/GooglePlay.js

```diff
@@ -227,29 +239,400 @@ function updateTrack(packageName, track, versionCode, userFraction) {
+ *    ├ tvScreenshots
+ *    |  └ *.png || *.jpg || *.jpeg
+ *    └ wearScreenshots
+ *       └ *.png || *.jpg || *.jpeg
+ * @param {string} directory Directory where the "images" folder is found matching the structure specified above
+ * @returns {Object} imageList Map of image types to lists of images matching that type.
+ *                              { [imageType]: string[] }
+ */
+function getImageList(directory) {
+    var imageTypes = ["featureGraphic", "icon", "promoGraphic", "tvBanner", "phoneScreenshots", "sevenInchScreenshots", "tenInchScreenshots", "tvScreenshots", "wearScreenshots"];
+    var acceptedExtensions = [".png", ".jpg", ".jpeg"];
+
+    var imageDirectory = path.join(directory, "images");
+    var imageList = {};
+
+    for (var i in imageTypes) {
+        var shouldAttemptUpload = false;
+        var imageType = imageTypes[i];
+
+        imageList[imageType] = [];
+
+        tl.debug(`Attempting to get images of type ${imageType}`);
+        switch (imageType) {
+            case "featureGraphic":
+            case "icon":
+            case "promoGraphic":
+            case "tvBanner":
+                for (var i = 0; i < acceptedExtensions.length && !shouldAttemptUpload; i++) {
+                    var fullPathToFileToCheck = path.join(imageDirectory, imageType + acceptedExtensions[i]);
+                    try {
+                        var imageStat = fs.statSync(fullPathToFileToCheck);
+                        if (imageStat) {
+                            shouldAttemptUpload = imageStat.isFile();
+                            if (shouldAttemptUpload) {
+                                tl.debug(`Found image for type ${imageType} at ${fullPathToFileToCheck}`);
+                                imageList[imageType].push(fullPathToFileToCheck);
+                                break;
+                            }
+                        }
+                    } catch (e) {
+                        tl.debug(`File ${fullPathToFileToCheck} doesn't exist. Skipping...`);
+                    }
+                }
+
+                if (!shouldAttemptUpload) {
+                    tl.debug(`Image for ${imageType} was not found. Skipping...`);
+                }
+                break;
+            case "phoneScreenshots":
+            case "sevenInchScreenshots":
+            case "tenInchScreenshots":
+            case "tvScreenshots":
+            case "wearScreenshots":
+                try {
+                    var fullPathToDirToCheck = path.join(imageDirectory, imageType);
+                    var imageStat = fs.statSync(fullPathToDirToCheck);
+                    if (imageStat) {
+                        tl.debug(`Found something for type ${imageType}`);
+                        shouldAttemptUpload = imageStat.isDirectory();
+                        if (!shouldAttemptUpload) {
+                            tl.debug(`Stat returned that ${imageType} was not a directory. Is there a file that shares this name?`);
```

Comment:
I have gone through and changed a couple more of the information message to console.log. On another note, as far as I can tell, if you ask VSTS for the full logs, it does contain all the debug messages.